### DISCUSSION TITLE
How can i set validation-inline file like as default validation lang?

### DISCUSSION THREAD

**Discussion by Luigidefra** [Laravel-Lang/common#43] [Q&A] [answered]
How can i set validation-inline file like as default validation lang?

**Comment by caouecs** [comment, marked as answer]
Hello,
set the name of validation-inline.php to validation.php

**Reply by andrey-helldar** [reply]
@Luigidefra, or you can use a translation manager ([andrey-helldar/laravel-lang-publisher](https://github.com/andrey-helldar/laravel-lang-publisher)) that will do it automatically.

In the config file posted to the app ([config/lang-publisher](https://github.com/andrey-helldar/laravel-lang-publisher/blob/main/config/public.php#L6-L19)), you will need to replace `'inline' => false` with `'inline' => true`.

**Reply by caouecs** [reply]
Ad-inside :angel:

**Reply by andrey-helldar** [reply]
Why not 😁
